modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/PodcastSyncProcess.kt

```diff
@@ -321,6 +325,21 @@ class PodcastSyncProcess(
             }
     }
 
+    private fun syncRatings(): Completable {
+        return rxCompletable {
+            syncManager.getPodcastRatings()?.podcastRatingsList
+                ?.mapNotNull { rating ->
+                    val modifiedAt = rating.modifiedAt.toDate() ?: return@mapNotNull null
+                    UserPodcastRating(
+                        podcastUuid = rating.podcastUuid,
+                        rating = rating.podcastRating,
+                        modifiedAt = modifiedAt,
+                    )
+                }
+                ?.let { ratingsManager.updateUserRatings(it) }
```

Comment:
Should we try to add this to the regular sync or include the greatest modifiedAt in the call so we don't download all the ratings with every refresh? Another option could be downloading the ratings before the end of year stats if that's the only time we use the table. I could be overthinking this and maybe it doesn't matter. 🤔

Reply:
> Should we try to add this to the regular sync or include the greatest modifiedAt in the call so we don't download all the ratings with every refresh?

Maybe? I don't know. It doesn't seem like a lot of data. Especially considering it is a proto call. But I don't think that using the greatest `modifiedAt` is correct. If I change a rating on a different device it might never update locally.

> Another option could be downloading the ratings before the end of year stats if that's the only time we use the table.

I do that using `EndOfYearSync` sync class. I think overall syncing and syncing for EoY are separate things and shouldn't be conflated. Syncing here allows us to migrate our rating process to not check a rating with a network call when opening the screen.